source/layouts/layout_pro.html.erb

```diff
@@ -1,19 +0,0 @@
-<!DOCTYPE html>
-<%
-  page_classes = locals[:page_classes] || current_page.data.page_classes
-  section_class = locals[:section_class] || current_page.data.section_class
-%>
-
-<html lang="<%= current_locale_obj[:lang] || 'en' %>" class="no-js">
-  <!-- pro layout -->
-  <%= partial "layouts/partials/head", locals: locals.merge({ pro: true }) %>
-
-  <% partial "layouts/partials/body", locals: locals.merge({
-    professional: true,
```

Comment:
Was there anything to remove around this `professional` and `enable_intercom`?

Is Intercom migrated to Gatsby?

Reply:
For `professional` we would still want to maintain the links changing to pro signup when on this page, so I don't think so. `enable_intercom` though I'm not so sure - what is intercom used for? 

I've found this, but not really sure what the widget is supposed to do, is it just a matter of having the configuration there? 
```
<% if locals[:enable_intercom] == true && ApplicationConfig.const_defined?(:Intercom) && ApplicationConfig::Intercom::APP_ID %>
  <script>
    window.intercomSettings = { app_id: '<%= ApplicationConfig::Intercom::APP_ID %>' };

    (function(){var w=window;var ic=w.Intercom;if(typeof ic==="function"){ic('reattach_activator');ic('update',intercomSettings);}else{var d=document;var i=function(){i.c(arguments)};i.q=[];i.c=function(args){i.q.push(args)};w.Intercom=i;function l(){var s=d.createElement('script');s.type='text/javascript';s.async=true;
    s.src='https://widget.intercom.io/widget/<%= ApplicationConfig::Intercom::APP_ID %>';
    var x=d.getElementsByTagName('script')[0];x.parentNode.insertBefore(s,x);}if(w.attachEvent){w.attachEvent('onload',l);}else{w.addEventListener('load',l,false);}}})()
  </script>
<% end %>
```

Reply:
I mean you've removed this page right?  Why would you want to maintain the links changing on this page if it's removed?

And Intercom was added like in the past 6-12 months by request of Adriana.  I expect Adriana scoped this out thoroughly–if not, ask her to 😬.  If they now want it removed, that's a bit weird (probably good, not sure how much effort it would be to add via React).

Reply:
haha 🤦 you're so right! 

sweet, I've flicked her a message about it